paddle/fluid/framework/ir/node.h

```diff
@@ -107,6 +109,28 @@ class Node {
   std::vector<Node*> inputs;
   std::vector<Node*> outputs;
 
+  template <typename AttrType>
```

Comment:
You can define a NodeWrapper and put all attributes (or support any) inside that NodeWrapper. See `WrappedBy` method.

Reply:
Will change in another PR, too big change here.